fw/http_parser.c

```diff
@@ -8741,10 +8746,9 @@ tfw_h2_parse_req(void *req_data, unsigned char *data, size_t len,
 
 	if (likely(type != HTTP2_DATA))
 		r = tfw_hpack_decode(&ctx->hpack, data, len, req, parsed);
-	else if (unlikely(req->method == TFW_HTTP_METH_GET
-		 || req->method == TFW_HTTP_METH_HEAD
-		 || req->method == TFW_HTTP_METH_DELETE
-		 || req->method == TFW_HTTP_METH_TRACE))
+	else if ((req->method_override
+			&& TFW_HTTP_IS_METH_BODYLESS(req->method_override))
+		    || TFW_HTTP_IS_METH_BODYLESS(req->method))
 	{
 		r = T_DROP;
 	}
```

Comment:
Could you please adjust coding style of the block as
```C
        if (likely(type != HTTP2_DATA)) {
                r = tfw_hpack_decode(&ctx->hpack, data, len, req, parsed);
        }
        else if ((req->method_override
                  && TFW_HTTP_IS_METH_BODYLESS(req->method_override))
                 || TFW_HTTP_IS_METH_BODYLESS(req->method))
        {
                r = T_DROP;
        }
        else {
                r = tfw_h2_parse_body(data, len, req, parsed);
        }
```

Reply:
Done